pkg/driver/executor/executor.go

```diff
@@ -383,42 +383,49 @@ func (ex *Executor) patchServerPortsForPodNetwork(serverID string) error {
 		return fmt.Errorf("failed to resolve network IDs for the pod network %v", err)
 	}
 
+	// coalesce all pod network CIDRs into a single slice.
+	podCIDRs := ex.Config.Spec.PodNetworkCIDRs
+	if ex.Config.Spec.PodNetworkCidr != "" {
+		podCIDRs = append(podCIDRs, ex.Config.Spec.PodNetworkCidr)
+	}
```

Comment:
You might end up with the pod range being in here twice until `PodNetworkCidr` is removed. This could lead to additional log lines below. However, filtering duplicates may complicate the code.
I leave it up for you to decide..